client/src/components/Input.js

```diff
@@ -0,0 +1,18 @@
+import styled from '@emotion/styled';
+import colors from '../utils/colors';
```

Comment:
Actually, this would be a good case for a theme.
But it works like this too, so it's up to you.